pkg/operator/ceph/pool/controller.go

```diff
@@ -385,13 +385,16 @@ func createPool(context *clusterd.Context, clusterInfo *cephclient.ClusterInfo,
 		return errors.Wrapf(err, "failed to create pool %q", p.Name)
 	}
 
-	logger.Infof("initializing pool %q", p.Name)
-	args := []string{"pool", "init", p.Name}
-	output, err := cephclient.NewRBDCommand(context, clusterInfo, args).Run()
-	if err != nil {
-		return errors.Wrapf(err, "failed to initialize pool %q. %s", p.Name, string(output))
+	if appName == poolApplicationNameRBD {
```

Comment:
Why this function is even called if it only needs to be run for `poolApplicationNameRBD`